network/definition/data/components/bmsb/bmsb-signals.yaml

```diff
@@ -211,3 +211,23 @@ signals:
   imdStatus:
     description: "Status of the BMSB IMD SFTY"
     discreteValues: "digitalStatus"
+
+  packCSPVoltage:
+    unit: 'V'
+    description: BMS Boss Pack Current Sensor Voltage
+    nativeRepresentation:
+      bitWidth: 8
+      range:
+        min: 0
+        max: 3
+    continuous: true
+
+  packCSNVoltage:
+    unit: 'V'
+    description: BMS Boss Pack Current Sensor Voltage
```

Comment:
is this meant to be the same description as the other one?

Reply:
good catch lil bro